src/libzerocoin/Accumulator.h

```diff
@@ -48,7 +48,7 @@ class Accumulator {
 	 **/
 	Accumulator(const AccumulatorAndProofParams* p, const CoinDenomination d);
 
-	Accumulator(const ZerocoinParams* p, const CoinDenomination d, Bignum bnValue = 0);
+	Accumulator(const ZerocoinParams* p, const CoinDenomination d, CBigNum bnValue = 0);
```

Comment:
This tab indent is whats causing the lint CI phase to fail. The file itself uses tabs instead of spaces for indentation (why god why 😩 ), however the linter only looks at lines that have been modified in the PR's commit(s).